extension/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@shopgate/products-add-properties",
```

Comment:
Please check the results of `npm audit`

Reply:
fixed. you do realize, this project has no actual (non-dev) dependencies?

Reply:
I do realise. When uploading to our system npm audit will be run and it makes no diff between type of a dependency.